packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -909,13 +910,17 @@ class _DraggableScrollableSheetScrollPosition extends ScrollPositionWithSingleCo
     }
   }
 
-  bool get _isAtSnapSize {
-    return extent.snapSizes.any((double snapSize) {
+  /// Checks if the sheet's current size is close to a snap size, returning the
+  /// snap size if so; returns null otherwise.
```

Comment:
Nit: I would make this 2 slashes instead of 3. Using 3 is for dart-doc-like comments.